### PR TITLE
simplify Ruby installation

### PR DESCRIPTION
**Pull request by afeld**
Install using `apt-get` rather than `rvm`. While less flexible, this has the advantage of:

* Being far simpler to install
* It will get upgraded with other system (particularly security) upgrades
* It's one less "package" manager to deal with

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -226,7 +226,7 @@ http://docs.ansible.com/
 ## Roadmap
 Cross platform compatibility (Ubuntu, CentOS)
 
-Greater diversity of interchangeable roles (nginx/apache, unicorn/puma/passenger, rvm/rbenv)
+Greater diversity of interchangeable roles (nginx/apache, unicorn/puma/passenger)
```
Comment (afeld) [inline README.md]: Is this actually something users are asking for?

File: roles/unicorn/tasks/main.yml [hunk 2]
```diff
@@ -34,7 +34,6 @@
   when: unicorn_process.stat.exists == True
 
 - name: Start Unicorn
-  shell: "/home/deploy/.rvm/gems/ruby-2.3.2/wrappers/unicorn -c config/unicorn.rb -E production -D"
+  shell: "bin/bundle exec unicorn -c config/unicorn.rb -E production -D"
```
Comment (afeld) [inline roles/unicorn/tasks/main.yml]: No binstub for Unicorn.

**Comment by voodoorai2000** [comment]
Interesting 🤔

The less dependencies the better, for sure!

However regarding Ruby versions, we do use [capistrano-rvm](https://github.com/rvm/rvm1-capistrano3) to make deployments and update the Ruby version accordingly.

Not all Ruby versions work right now with CONSUL, so we have to be careful with the versions installed.

I think having RVM installed by the installer, will help users to upgrade to the next Ruby versions when CONSUL has been verified to work on them and updated accordingly in the [capistrano-rvm configuration option](https://github.com/consul/consul/blob/master/config/deploy.rb#L10).
 
I see the Installer as a one time task, and then using Capistrano for day to day deployments. However I hadn't considered seriously system library updates. I assumed system administrators would deal with those. Correct me if you feel there is a better way to approach it 🙏

File: README.md (hunk 1 above)
Comment (voodoorai2000) [reply README.md]: Not really 😌

File: roles/unicorn/tasks/main.yml (hunk 2 above)
Comment (voodoorai2000) [reply roles/unicorn/tasks/main.yml]: Nice 👌

**Comment by afeld** [comment]
> Not all Ruby versions work right now with CONSUL, so we have to be careful with the versions installed.

Ok, this is a good reason to stick with RVM. Was running into problems with the role, but will try updating it and seeing if I still hit them. Thanks!

**Pull request closed without merging**